README.md

```diff
@@ -15,7 +15,7 @@ Comes with a naming system, for developerability.
 Written in Erlang.
 
 To install and run the Aeternity node, see the instructions [below](#how-to-start)
-or just [follow the progress of the project in Pivotal Tracker][pivotal].
+or just follow the progress of the project via GitHub Issues ([Deprecated Project Management: Pivotal Tracker][pivotal]).
```

Comment:
Its not just deprecated, but simply not supported anymore. So it shouldn't be linked here.

Reply:
Great! Thank you Tino :)